package.json

```diff
@@ -11,16 +11,16 @@
     "test": "mocha --compilers ts:ts-node/register,tsx:ts-node/register --recursive test",
     "watch:test": "mocha -w --compilers ts:ts-node/register,tsx:ts-node/register --recursive test"
   },
-  "author": "Renke Grunwald <renke@renke.org> (https://github.com/renke)",
+  "author": "Unsplash <dev@unsplash.com> (https://unsplash.com)",
   "repository": "unsplash/import-sort-style-unsplash",
   "license": "ISC",
   "devDependencies": {
     "@types/chai": "^4.0.0",
     "@types/mocha": "^2.2.41",
     "@types/node": "^7.0.31",
     "chai": "^4.0.2",
-    "import-sort-parser": "^4.2.0",
-    "import-sort-style": "^4.2.0",
+    "import-sort-parser": "^6.0.0",
+    "import-sort-style": "^6.0.0",
```

Comment:
Not sure if we're still planning to upgrade `import-sort`. If we decide not to do that then we might want to revert this PR?

Reply:
Apologies. I've rebased on master directly 😅 https://github.com/unsplash/import-sort-style-unsplash/commits/master